app/templates/src/main/java/package/security/jwt/_TokenAuthenticationService.java

```diff
@@ -0,0 +1,41 @@
+package <%=packageName%>.security.jwt;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.stereotype.Service;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * Created by jhipster on 10/25/15.
+ */
+@Service
+public class TokenAuthenticationService {
+
+    private static final String AUTH_HEADER_NAME = "X-AUTH-TOKEN";
```

Comment:
We need to discuss this -> the "X-" should now not be used (latest RFC...) and as this is specific to JHipster I'd do something like "X-JHIPSTER-TOKEN" or "X-JHIPSTER-AUTH"